src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/request/CaseWorkersProfileCreationRequest.java

```diff
@@ -34,20 +34,26 @@
 
     private List<CaseWorkerWorkAreaRequest> workerWorkAreaRequests;
     private long rowId;
+    private boolean caseAllocator;
+    private boolean taskSupervisor;
 
     @JsonCreator
     public CaseWorkersProfileCreationRequest(@JsonProperty("first_name") String firstName,
-            @JsonProperty("last_name") String lastName,
-            @JsonProperty("email_id") String emailId,
-            @JsonProperty("region_id") Integer regionId,
-            @JsonProperty("user_type") String userType,
-            @JsonProperty("region") String region,
-            @JsonProperty("suspended") boolean suspended,
-            @JsonProperty("idam_roles") Set<String> idamRoles,
-            @JsonProperty("roles") List<CaseWorkerRoleRequest> roles,
-            @JsonProperty("base_location") List<CaseWorkerLocationRequest> baseLocations,
-            @JsonProperty("work_area") List<CaseWorkerWorkAreaRequest> workerWorkAreaRequests,
-            @JsonProperty("rowId") long rowId) {
+                                             @JsonProperty("last_name") String lastName,
```

Comment:
(Just a suggestion may considered optional) @Getter, @Setter can be used here instead of parameterised constructor as in future the parameters may increase will be difficult to maintain. Sonar doesn't accept more that 7 params anyways.

Reply:
We already have `@Getter @Setter`. Will remove this explicit constructor as it is being used only in tests. thanks for pointing out